wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,273 @@
+It is possible to [enable custom overrides][Custom Sample Overrides] (please put hit sounds files inside the beatmap's folder first) for custom hit sounds instead.
 
-### Approximate Difficulty Rating
+#### Sample Set Volume
 
-This is a summary of all of the settings chosen on this page. More stars mean harder maps and more [score](/wiki/Score). This is not the final star ranking of the song; it is just an approximation based on the settings you chose.
+Set the volume of the sample set.
 
-Audio
------
+#### Test Sample Set
 
-![Audio section](img/SS_Audio.jpg "Audio section")
+Click on the buttons to test out how the sample set sounds. Four buttons are provided, namely: `Normal`, `Whistle`, `Finish`, and `Clap`.
 
-### Default Sample Settings
+These sounds are additive; `Whistle` is a combination of `Normal` and `Whistle`.
 
-Here you can configure the hit sounds to your liking.
+### Misc. Toggles
 
-If you have [timing](/wiki/Beatmap_Editor/Timing) sections (F6) that change either the hit-sound set or volume, you will not be able to adjust them here. You can click reset settings to remove them. Most mappers use timing sections to set the settings you can find here.
+Miscellaneous toggles.
 
-#### Sample Set Selection
+`Samples match playback rate (for fully-hitsounded maps)` checkbox will adjust the hit sounds' pitch accordingly to the speed modifiers. Unchecking it will let the hit sounds play at its own preset pitch. This option is subjective to the mapper so please use discretion before checking it.
 
-Here you can choose whether to use the Normal, Soft, or Drum built-in sample set. You can also [enable custom overrides](/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides) (Put hitsounds files inside the map's folder first).
+## Colours
 
-#### Sample Set Volume
+![Colours section][SS_colours]
 
-Here you can set the volume of the sample set. This is important, because while you want to hear the hit sounds (an important aspect of osu! and osu!mania), you do not want the hit-sounds to drown out the actual song.
+Configure combo colours and playfield's background colour.
 
-#### Test Sample Set
+### Hitcircle/Slider combos
 
-Click on the buttons to test out how the sample sounds. These sounds are additive, Whistle is a combination of Normal and Whistle.
+Configure the custom combo colour set.
 
-Colours
--------
+Combo colours are an important part of the beatmap's aesthetic value.
 
-![Colours section](img/SS_Colours.jpg "Colours section")
+Clicking a combo colour or clicking on `New Combo Colour` opens up the OS's color picker (as shown in the image, uploader used a Windows OS). Choose colors that compliment the background but the colours must not camouflage into the background.
```

Comment:
OS's -> OS'
color -> colour

uploader used a Windows OS -> it opens Windows' colour picker)
colors -> colours